docs/whats-new.rst

```diff
@@ -1,6 +1,12 @@
 What's New
 ==========
 
+v0.1.1
+------
+- Bug fixes related to the start of the ``calc.py`` refactor in :issue:`90`
+  (see :issue:`126` and :issue:`128`)
+- Compatability with xarray version 0.9.0
+
```

Comment:
In looking over xarray's and a few others' What's New sections, I think we should be more explicit:

```
Enhancements
~~~~~~~~~~~~
- Support for xarray 0.9.0.  By `Spencer Clark <https://github.com/spencerkclark>`_.

Bug fixes
~~~~~~~~~
- Fix an instance where the name for pressure half levels was mistakenly replaced with the name for pressure full levels (:issue:`126`).  By `Spencer Clark <https://github.com/spencerkclark>`_.
- Etc. for each other bug fixed.
```